lib/consumer.ex

```diff
@@ -45,6 +45,10 @@ defmodule GenRMQ.Consumer do
   the queue has not been redeclared, and basic.get has not been invoked
   for a duration of at least the expiration period
 
+  `queue_max_priority` - defines if a declared queue should be a priority queue.
+  Should be set to a value from 0..255 range. If it is greater than 255, queue
+  max priority will be set to 255.
```

Comment:
Let's include the [recommendation](https://www.rabbitmq.com/priority.html): "Values between 1 and 10 are recommended."